### ISSUE TITLE
Add support for a reduced motion prop

### ISSUE THREAD

**Issue by Sandstedt** [mojs/mojs#216]

**Comment by xavierfoucrier** [comment]
Hi @Sandstedt!

Can you give a short code sample to figure out what you mean? :wink:

**Comment by Sandstedt** [comment]
Here's and article about what it is and when it should be used: https://developers.google.com/web/updates/2019/03/prefers-reduced-motion

```js
const opacityAnim = new mojs.Shape({
  opacity: {0 : 1},
  a11y: false
});
```

Not sure about the prop name. Should be clear that it has with accesability to do and that we want to ignore the prefere reduced motion check.

**Comment by Sandstedt** [comment]
So if the `a11y` prop in the example above is set to `true`, the animation wont play if the user has set "Prefere reduced motion" in their system settings.

**Comment by xavierfoucrier** [comment]
Thanks for the clarification, it could be a great feature: respect the user preference is more important than ever today.

**Comment by Sandstedt** [comment]
Yeah, it very easy to do self as a developer, by adding the stop method inside the media query. But then the developer need to be aware of it. Better to have them opt-out instead if they know what they're doing.
